storage/index_insert_queue.go

```diff
@@ -48,8 +46,9 @@ const (
 )
 
 var (
-	defaultIndexBatchBackoff   = time.Second
-	defaultIndexPerSecondLimit = 10000
+	// TODO(prateek): undo this stuff
+	defaultIndexBatchBackoff   = time.Millisecond
```

Comment:
i say leave this around for now. they're sensible enough defaults. I'll come back and wire up to runtime/config in https://github.com/m3db/m3db/pull/604